src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
```

Comment:
The fields 'firstLeg' and 'secondLeg' should not be declared as 'final' because in the future you may want to change their values (for example, for scaling the figure).

Reply:
done